.ci/pytorch/win-test.sh

```diff
@@ -35,7 +35,7 @@ if [[ "$BUILD_ENVIRONMENT" == *cuda* ]]; then
 fi
 
 # TODO: Move both of them to Windows AMI
-python -m pip install pytest-rerunfailures==10.3 pytest-cpp==2.3.0 tensorboard==2.13.0
+python -m pip install pytest-rerunfailures==10.3 pytest-cpp==2.3.0 tensorboard==2.13.0 pytest-subtests==0.13.1
```

Comment:
Now that you mention Windows, I think you will need to add this to https://github.com/pytorch/pytorch/blob/main/.github/requirements/pip-requirements-macOS.txt for mac

Reply:
done, thanks!